client/lib/after-layout-flush/test/index.js

```diff
@@ -144,9 +142,14 @@ describe( 'afterLayoutFlush', () => {
 	} );
 
 	describe( 'in browsers that do not support requestAnimationFrame', () => {
+		let originalRequestAnimationFrame;
 		beforeAll( () => {
-			sinon.restore();
-			sinon.stub( window, 'requestAnimationFrame' ).value( undefined );
+			originalRequestAnimationFrame = window.requestAnimationFrame;
+			window.requestAnimationFrame = undefined;
```

Comment:
I wish something like
```
jest.spyOnProp( window, 'requestAnimationFrame' ).mockValue( undefined );
```
was possible, but it isn't. Not without an extension package. `jest.spyOn` works only with functions.

Reply:
Yeah, exactly. That's why I mocked it in such an old-school way 😉